driver-legacy/src/main/com/mongodb/DBCursor.java

```diff
@@ -335,7 +335,10 @@ public DBCursor maxTime(final long maxTime, final TimeUnit timeUnit) {
      * @return a {@code DBObject} containing the explain output for this DBCursor's query
      * @throws MongoException if the operation failed
      * @mongodb.driver.manual reference/explain Explain Output
+     * @deprecated This method is no longer part of our CRUD API. In 4.x this method will only support MongoDB 3.2 and up (where the
```

Comment:
No need to mentions 4.x if the deprecation is happening in 4.0

Reply:
I have altered the deprecation comment to not mention 4.x.